en/appendices/5-0-migration-guide.rst

```diff
@@ -119,6 +119,12 @@ ORM
   to retain the previous behavior.
 - ``allowMultipleNulls`` option for ``isUnique`` rule now default to true matching
   the original 3.x behavior.
+- ``Table::query()`` has been removed. Instead operation focused methods have
```

Comment:
`Table::query()` hasn't been removed. I wanted to rename it to `selectQuery()` but couldn't because of `RepositoryInterface::query()`. Do we want to update the interface.

Reply:
Hmm changing the interface will spill out into other plugins that are using the `RepositoryInterface` as well. I don't know if we get much benefit from breaking more plugins.